services/src/main/java/org/keycloak/keys/loader/PublicKeyStorageManager.java

```diff
@@ -54,6 +55,20 @@ public static KeyWrapper getClientPublicKeyWrapper(KeycloakSession session, Clie
         return keyStorage.getPublicKey(modelKey, kid, loader);
     }
 
+    public static KeyWrapper getClientPublicKeyWrapper(KeycloakSession session, ClientModel client, JWK.Use keyUse) {
```

Comment:
I think this method can be removed?

Reply:
Yes, this method has already not been used. I'll remove it.